posts/month-in-mathlib-aug-2021.md

```diff
@@ -0,0 +1,74 @@
+---
+author: 'Mathlib community'
+category: 'month-in-mathlib'
+date: 2021-08-31 08:56:51+00:00
+description: ''
+has_math: true
+link: ''
+slug: month-in-mathlib-aug-2021
+tags: ''
+title: This month in mathlib (Aug 2021)
+type: text
+---
+
+# This month in mathlib (Aug 2021)
+
+## Highlighted PRs
+
+* [PR #8652](https://github.com/leanprover-community/mathlib/pull/8652) :: chore(*): update lean to 3.32.1
+
+  The community fork of Lean made two new
+  [releases](https://github.com/leanprover-community/lean/blob/master/doc/changes.md#3321c-12-august-2021)
+  `3.32.0` and `3.32.1`.
+  This is part of the preparations for porting mathlib to Lean 4.
+
+* [PR #8281](https://github.com/leanprover-community/mathlib/pull/8281) :: feat(topology,geometry/manifold): continuous and smooth partition of unity
+
+  See the [companion blogpost](continuous-partitions-of-unity/) for details.
+
+* Radon-Nikodym and Lebesgue decomposition. The four PRs
+  [PR #8645](https://github.com/leanprover-community/mathlib/pull/8645)
+  [PR #8687](https://github.com/leanprover-community/mathlib/pull/8687)
+  [PR #8763](https://github.com/leanprover-community/mathlib/pull/8763)
+  [PR #8875](https://github.com/leanprover-community/mathlib/pull/8875)
+  together contribute
+  the Lebesgue decomposition for sigma-finite measures
+  and the Radon-Nikodym theorem.
+
+* [PR #7978](https://github.com/leanprover-community/mathlib/pull/7978) :: feat(measure_theory/interval_integral): strong version of FTC-2
+
+  This PR signals the start of a serious attack on Stokes' theorem,
+  which will unlock a general approach to complex analysis.
+
+* [PR #8692](https://github.com/leanprover-community/mathlib/pull/8692) :: feat(field_theory): finite fields exist
+
+  Most of this PR had been lying around for ages,
+  but it was finally put together in mathlib.
+  It shows the existence and uniqueness of finite fields.
+
+* Among several PRs from the Dedekind project, the two most noteworthy are
```

Comment:
Is there something we can link to for the "Dedekind project"?

Reply:
@Vierkantor would be the best person to ask

Reply:
We use
```suggestion
* Among several PRs from the [Dedekind project](https://github.com/lean-forward/class-number), the two most noteworthy are
```